### DISCUSSION TITLE
Using two or more graphics cards at the same time

### DISCUSSION THREAD

**Discussion by TimmekHW** [lshqqytiger/stable-diffusion-webui-directml#391] [Ideas]
Currently, the original Stable Diffusion can only use a single graphics card.

But with DirectML we can in theory combine several graphics cards into 1 and use them all to speed up computation.

**Comment by lshqqytiger** [comment]
https://github.com/microsoft/DirectML/issues/443

**Reply by TimmekHW** [reply]
that is, due to the lack of support in DirectML there will not be yet?

**Reply by lshqqytiger** [reply]
torch-directml does not support any data parallelism techniques and even it can't move tensors directly between gpus.
